src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -96,7 +96,7 @@ private void testContent() throws Exception {
 
         // Verify with retry after upload picture due to inconsistency of .click in detecting page load
         profilePage.verifyStatusWithRetry(Const.StatusMessages.STUDENT_PROFILE_PICTURE_SAVED, 10);
-        profilePage.verifyHtmlPart(By.className("profile-pic-edit-toolbar"), "/studentProfilePictureModalEdit.html");
+        profilePage.verifyHtml("/studentProfilePageFilled.html");
```

Comment:
I think main content check is sufficient. I don't see anything important outside the main content being tested here.